packages/salesforcedx-vscode-core/src/context/workspaceOrgType.ts

```diff
@@ -10,34 +10,58 @@ import * as vscode from 'vscode';
 
 import { ForceConfigGet } from '@salesforce/salesforcedx-utils-vscode/out/src/cli';
 
-export async function setupWorkspaceOrgType() {
+export enum OrgType {
+  SourceTracked,
+  NonSourceTracked
+}
+
+export async function getWorkspaceOrgType(): Promise<OrgType> {
```

Comment:
I pulled out this getWorkspaceOrgType method so that I could use it inside of pushOrDeployOnSave.ts